pkg/segment/results/segresults/segresults.go

```diff
@@ -368,6 +368,26 @@ func (sr *SearchResults) UpdateSegmentStats(sstMap map[string]*structs.SegStats,
 			}
 			sort.Strings(uniqueStrings)
 			strVal := strings.Join(uniqueStrings, "&nbsp")
+			sr.segStatsResults.measureResults[measureAgg.String()] = utils.CValueEnclosure{
+				Dtype: utils.SS_DT_STRING,
+				CVal:  strVal,
+			}
+			continue
+		case utils.List:
+			strList := make([]string, 0, 100)
```

Comment:
nit: It looks like you only return at most 100 because that's what the splunk documentation says:
> If more than 100 values are in the field, only the first 100 are returned.

It would be good to add a comment here, saying something like that, just to clarify to future readers of this code where the number 100 came from

Reply:
Added the comment in the latest commit.